lib/v2/modrinth/versions.js

```diff
@@ -19,32 +20,28 @@ module.exports = async (
     const {
         /** @type string */
         id,
+        /** @type {string | undefined} */
+        routeParams,
     } = ctx.params;
-    let {
-        /** @type {string | string[] | undefined} */
-        loaders,
-        /** @type {string | string[] | undefined} */
-        game_versions,
-    } = ctx.query;
-    const { featured } = ctx.query;
 
-    if (typeof loaders === 'string') {
-        loaders = [loaders];
-    }
-    if (typeof game_versions === 'string') {
-        game_versions = [game_versions];
-    }
+    /**
+     * /@type {{
+     *   loaders: string | string[] | undefined
+     *   game_versions: string | string[] | undefined
+     *   featured: string | undefined
+     * }}
+     */
+    const parsedQuery = new URLSearchParams(routeParams);
+
+    parsedQuery.set('loaders', JSON.stringify(parsedQuery.getAll('loaders')));
+    parsedQuery.set('game_versions', JSON.stringify(parsedQuery.getAll('game_versions')));
```

Comment:
```suggestion
    parsedQuery.set('loaders', parsedQuery.has('loaders') ? JSON.stringify(parsedQuery.getAll('loaders')) : '');
    parsedQuery.set('game_versions', parsedQuery.has('game_versions') ? JSON.stringify(parsedQuery.getAll('game_versions')) : '');
```
IMHO if `loaders` or `game_versions` aren't set, the route should return results with no filtering instead of filtering everything.

Reply:
xd